RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -105,17 +105,16 @@
                      'lowPtQuadStepTracks',
                      'lowPtTripletStepTracks',
                      'detachedQuadStepTracks',
-                     'pixelPairStepTracks'],
-    hasSelector = [1,1,1,1,1,1],
+                    ],
+    hasSelector = [1,1,1,1,1],
     indivShareFrac = [1.0,0.16,0.095,0.09,0.09,0.09],
```

Comment:
should the indivShareFrac length be changed to 5 as well? (I'm assuming it's cosmetic, given that the code ran; no length match asserts)